app/src/main/java/amritansh/tripathi/com/searchflix/presentation/MainActivity.kt

```diff
@@ -68,16 +72,15 @@ class MainActivity : DaggerAppCompatActivity() {
         compositeDisposable.dispose()
     }
 
-    //TODO:1)Add Fragment Navigator
-    //TODO:2)Add backstack management
-    //TODO:3)Add Loading progress bar
-    //TODO:4)Add pagination
-    //TODO:5)Add delegate Adapter
-    //TODO:6)Add Recommendations in Details
-    //TODO:7)Add UserManagement
-    //TODO:8)ADD Test Cases
-    //TODO:9)Add User Prefrences
-    //TODO:10)Add Room
-    //TODO:11)Add empty states
+    //TODO:1)Add backstack management
+    //TODO:2)Add Loading progress bar
+    //TODO:3)Add pagination
+    //TODO:4)Add delegate Adapter
+    //TODO:5)Add Recommendations in Details
+    //TODO:6)Add UserManagement
+    //TODO:7)ADD Test Cases
+    //TODO:8)Add User Prefrences
+    //TODO:9)Add Room
+    //TODO:10)Add empty states
```

Comment:
I think we should extract those to issues.

Reply:
@tripathiamritansh can you do that?